pytest_fixtures/rh_cloud.py

```diff
@@ -30,10 +30,12 @@ def organization_ak_setup(module_manifest_org):
         environment=entities.LifecycleEnvironment(id=module_manifest_org.library.id),
         auto_attach=True,
     ).create()
-    subscription = entities.Subscription(organization=module_manifest_org).search(
+    subscription = entities.Subscription(organization=module_manifest_org)
+    subscription.refresh_manifest(data={'organization_id': module_manifest_org.id})
```

Comment:
Hi, you cannot refresh a cloned manifest and refreshing the normal manifest can be destructive to other tests.

Reply:
This PR code is not relevant to issue anymore. The test is failing because of other reason. 
I will debug the issue and add proper fix soon.